src/components/EmailNotifications/VerifyEmailAddress.js

```diff
@@ -0,0 +1,258 @@
+                      value={emailAddress}
+                      adornment={
+                        emailInvalid === false ? (
+                          <IconCheck
+                            css={`
+                              color: ${theme.positive};
+                            `}
+                          />
+                        ) : emailAddress.trim() ? (
+                          <IconCross
+                            css={`
+                              color: ${theme.negative};
+                            `}
+                          />
+                        ) : (
+                          <IconCheck
+                            css={`
+                              opacity: 0;
+                            `}
+                          />
+                        )
+                      }
+                      adornmentPosition="end"
+                      type="email"
+                      wide
+                      onChange={handleEmailAddressChange}
+                      placeholder="email@address.com"
+                      onBlur={handleEmailAddressBlur}
+                    />
+                    {emailInvalid && (
+                      <p
+                        css={`
+                          color: ${theme.negative};
+                          ${textStyle('body4')};
+                          float: left;
+                          margin-top: ${0.5 * GU}px;
+                        `}
+                      >
+                        Please enter a valid email address.
+                      </p>
+                    )}
+                  </Field>
+                </div>
+                <LegalTermsAndPolicy
+                  termsAccepted={termsAccepted}
+                  setTermsAccepted={setTermsAccepted}
+                  theme={theme}
+                />
+                <div
+                  css={`
+                    display: flex;
+                    justify-content: space-between;
+                    flex-direction: ${compactMode ? 'column' : 'row'};
+                    width: 100%;
+                    margin-bottom: ${1.5 * GU}px;
+                    margin-top: ${3 * GU}px;
+                  `}
+                >
+                  <ActionButtons
+                    compactMode={compactMode}
+                    onClick={onDeleteEmail}
+                    width={buttonWidth}
+                  >
+                    Delete email
+                  </ActionButtons>
+                  <ActionButtons
+                    compactMode={compactMode}
+                    mode="strong"
+                    disabled={emailInvalid || !termsAccepted}
+                    onClick={handleSubscribe}
+                    width={buttonWidth}
+                  >
+                    Send verification email
+                  </ActionButtons>
+                </div>
+              </>
+            )
+          }
+          return (
+            <Button
+              css={`
```

Comment:
Nitpick: we usually leave the `css` prop last, as it's usually noisy and doesn't impact the component's primary behaviour (through props) :)